aggregator/src/tests/proof_aggregation.rs

```diff
@@ -0,0 +1,155 @@
+use std::{fs, path::Path, process};
+
+use ark_std::test_rng;
+use halo2_proofs::{dev::MockProver, halo2curves::bn256::Fr, poly::commitment::Params};
+use itertools::Itertools;
+use snark_verifier::loader::halo2::halo2_ecc::halo2_base::utils::fs::gen_srs;
+use snark_verifier_sdk::{
+    gen_pk,
+    halo2::{gen_snark_shplonk, verify_snark_shplonk},
+    CircuitExt,
+};
+
+use crate::{AggregationCircuit, ChunkHash, CompressionCircuit};
+
+use super::mock_chunk::MockChunkCircuit;
+
+const CHUNKS_PER_BATCH: usize = 2;
+
+#[test]
+fn test_aggregation_circuit() {
```

Comment:
i think this test will run long time and needs much memory?   Is it included in CI by default? if yes, then we may better exclude it by adding `#[ignore="it takes too much time"]`.